torch/onnx/__init__.py

```diff
@@ -29,16 +43,22 @@ class CheckerError(Exception):
 
 
 class SymbolicContext:
-    r"""Provides extra context for symbolic functions.
+    """Extra context for symbolic functions.
 
-    Args:
-        params_dict (Dict[str, _C.IValue]): Mapping from graph initializer name to IValue.
-        env (Dict[_C.Value, _C.Value]): Mapping from Torch domain graph Value to ONNX domain graph Value.
-        cur_node (_C.Node): Current node being converted to ONNX domain.
-        onnx_block (_C.Block): Current ONNX block that converted nodes are being appended to.
+    Attributes:
+        params_dict: Mapping from graph initializer name to IValue.
```

Comment:
Can you keep the attribute type in docstring? PyTorch follows this https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
hmm it does look like the pytorch doc doesn't capture the param types https://pytorch.org/docs/stable/generated/torch.nn.Linear.html. This is unfortunate.

Reply:
I opened #78311

Reply:
Done